pylint_odoo/checkers/no_modules.py

```diff
@@ -371,11 +377,22 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def replace_and_first_lower(self, words):
+        """Convert 'Hello World' to 'hello_World'
```

Comment:
The purpose of converting 'Hello World' to 'hello_World' is so the programer can label a field named "hello_world" as "**H**ello **W**orld" with both words capitalized. The default label provided by odoo only capitalizes the first word.